core/test/network-records-to-devtools-log.js

```diff
@@ -244,6 +244,8 @@ function getRequestWillBeSentEvent(networkRecord, index, normalizedTiming) {
       frameId: networkRecord.frameId || `${idBase}.1`,
       redirectResponse: networkRecord.redirectResponse,
     },
+    targetType: networkRecord.sessionTargetType,
```

Comment:
this is required so should have a default?

Reply:
I left that out because otherwise `NetworkRecorder.recordsFromLogs` would need to do something similar (otherwise the roundtrip tests would fail), and it seemed better to not let such defaults bleed into non-test code.

Reply:
We want to start relying on this property though, and we don't want to manually add `sessionTargetType` to a bunch of tests every time a computed artifact or whatever starts using it.

Ideally #15005 will eventually get us to a place where test devtoolslogs are all updated. What about going wild and doing `targetType: 'sessionTargetType' in networkRecord ? networkRecord.sessionTargetType : 'page'`? That would default to including it, except for when constructed from real network records.

Reply:
going hog wild works, thanks :)